### PR TITLE
feat(console): cloud preview about page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- Add the about page to the cloud preview
- Update the welcome page's `onNext` logic
- Extract commonly used `.content` style into `layout.module.scss`
- Support custom option styles in `CardSelector`s.

## Testing

image

### CODE REVIEW

File: packages/console/src/App.tsx [hunk 1]
```diff
@@ -51,6 +51,7 @@ import AppContent from './containers/AppContent';
 import ApiResourcePermissions from './pages/ApiResourceDetails/ApiResourcePermissions';
 import ApiResourceSettings from './pages/ApiResourceDetails/ApiResourceSettings';
 import CloudPreview from './pages/CloudPreview';
+import CloudPreviewAbout from './pages/CloudPreview/pages/About';
```
Comment (wangsijie) [inline packages/console/src/App.tsx:54]: Can you create an issue to track: change "cloud preview" to something else like "CloudWelcome" or "CloudGuide"

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/pages/CloudPreview/pages/About/options.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import { CompanySize, Reason, Title } from '../../types';
+
+export const titleOptions: SelectorOption[] = [
+  { title: 'cloud_preview.about.title_developer', value: Title.Developer },
+  { title: 'cloud_preview.about.title_team_lead', value: Title.TeamLead },
+  { title: 'cloud_preview.about.title_ceo', value: Title.Ceo },
+  { title: 'cloud_preview.about.title_cto', value: Title.Cto },
+  { title: 'cloud_preview.about.title_product', value: Title.Product },
+  { title: 'cloud_preview.about.title_others', value: Title.Others },
```
Comment (charIeszhao) [inline packages/console/src/pages/CloudPreview/pages/About/options.tsx:11]: nit: you can put these in a nested object called `title`, so the structure will be better organized. For example in the phrase json you can use:

```
title: {
  developer: 'Developer',
  team_lead: 'Team Lead',
  ...,
}
```

File: packages/phrases/src/locales/ko/translation/admin-console/general.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ const general = {
   placeholder: '플레이스홀더',
   skip: '건너뛰기',
   next: '다음',
+  back: 'Back', // UNTRANSLATED
```
Comment (charIeszhao) [inline packages/phrases/src/locales/ko/translation/admin-console/general.ts:5]: nit: I think we'd better translate these phrases for cloud version, as this is not just for open source any more.

Comment (xiaoyijun) [reply packages/phrases/src/locales/ko/translation/admin-console/general.ts:5]: An issue added: LOG-5417

File: packages/console/src/App.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/App.tsx:54]: Added, LOG-5418

File: packages/console/src/pages/CloudPreview/pages/About/options.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/CloudPreview/pages/About/options.tsx:11]: Add an issue to track this.
LOG-5419

**Pull request merged**